nss-cohort-36/keahua-arboretum-net-surfers
# added random generator to feed functionality and overrode on several a…

…nimals

# Description

added clear function to new_feed_animal folder and added random generator to feed to decline prey 20% of the time.

Closes #[Enter issue #]

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jl-feed-random`
4. Create biomes and animals of your choice and feed until new message appears.  Special treatment to dolphins, spider, and kikakapu.  Also see if any clear statements missing

# Checklist:
